sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/SampleOverviewView.groovy

```diff
@@ -52,9 +51,9 @@ class SampleOverviewView extends VerticalLayout implements HasHotbar, HasTitle,
     private static Grid<Sample> createSamplesGrid(Collection<Sample> samples) {
         Grid<Sample> samplesGrid = new Grid<>()
         ListDataProvider<Sample> dataProvider = ListDataProvider.ofCollection(samples)
-        samplesGrid.addColumn(Sample::getCode).setCaption("Sample Code").setId("SampleCode")
         samplesGrid.addColumn(Sample::getName).setCaption("Sample Name").setId("SampleName")
         samplesGrid.addColumn(Sample::getStatus).setCaption("Sample Status").setId("SampleStatus").setStyleGenerator({Sample sample -> determineColor(sample.status)})
+        samplesGrid.addColumn(Sample::getCode).setCaption("Sample Code").setId("SampleCode")
```

Comment:
I think this change might not be necessary. Did you run it? Maybe we can talk about it over a quick call.

Reply:
what do you mean with it might not be necessary? I only moved the column to the right 😄